src/api/app/views/webui2/webui/search/_results_owner.html.haml

```diff
@@ -0,0 +1,41 @@
+.mt-4#search-results
+  %h4 Results:
+  - results.each do |result|
+    .search_result.mt-3
+      %h6
+        Responsible for package
```

Comment:
It's like this because of my local setup. I put the root project as the same project containing the package.

In production, it is more like:

`Responsible for package bat for openSUSE set in package utilities / bat`

Reply:
I will remove the first part, so it would be `Responsible for package utilities / bat`.